### ISSUE TITLE
[a11y] Form should have correct semantics role

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#161628]
when using the widget https://api.flutter.dev/flutter/widgets/Form-class.html, is should have the following role

| OS | role |
|--------|--------|
| web |form |
| ios | - |
| macos |  NSAccessibilityGroupRole, AXLandmarkForm |
| windows | - |
| android | - |
